src/components/CustomUrqlClient.js

```diff
@@ -1,12 +1,16 @@
-import { dedupExchange, cacheExchange, fetchExchange } from "@urql/core";
+import { cacheExchange, dedupExchange, fetchExchange } from "@urql/core";
 import { withUrqlClient } from "next-urql";
 import { refreshToken } from "src/lib/auth";
 import { authExchange } from "src/lib/authTokenExchange";
 export const withCustomUrqlClient = (Component) =>
   withUrqlClient(
     (ctx) => {
+      const url = ctx?.req
+        ? `http://localhost:${process.env.PORT}/api/graphql`
```

Comment:
Will it be enough for e2e testing between Docker containers(in the CI)?

Reply:
it's a hack :( 
On serverside, the fetch client need absolute url wich is not the case of client fetch